src/css-token-stream-value.js

```diff
@@ -16,16 +16,32 @@
   }
   internal.inherit(CSSTokenStreamValue, CSSStyleValue);
 
-  CSSTokenStreamValue.prototype.keys = function() {
-    return internal.objects.iterator(this._listOfReferences, function(key, value) { return key; });
+  CSSTokenStreamValue.prototype[Symbol.iterator] = function() {
```

Comment:
I could make the declaration of iterator support gated on the existence of Symbol though?